api/test/sql/UHT/test_schema.sql

```diff
@@ -319,6 +372,35 @@ insert into hbinccode (code, to_date, descrip1) values ('YTS', '2099-12-31', 'Mu
 insert into hbinccode (code, to_date, descrip1) values ('YTS', '2099-12-31', 'Face to face responsive architecture');
 insert into hbinccode (code, to_date, descrip1) values ('YTS', '2099-12-31', 'Innovative tertiary monitoring');
 
+--------------------
+
+CREATE TABLE [hbclaimnotes] (
+    [string_id] int,
+    [row_sequence] int,
+    [text_total] int,
+    [text_value] nvarchar(1786)
+);
+
+insert into hbclaimnotes (string_id, row_sequence, text_total, text_value) values (1, 1, 8125,  "User Id: aaaaaaa  Date: 19.03.2010 12:08:59  000000000
+ctax discrepancies report...assessment classification is correct.
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
loving the. accurate representation of our unique data <3

Reply:
ikr :D